tcg/tcg-op.c

```diff
@@ -2821,6 +2821,7 @@ static inline void plugin_gen_mem_callbacks(TCGv_cap_checked_ptr vaddr, uint16_t
 #ifdef TARGET_CHERI
 static inline void gen_cheri_invalidate_tags(TCGv_cap_checked_ptr out_addr, TCGv_i32 memop) {
     gen_helper_cheri_invalidate_tags(cpu_env, out_addr, memop);
+    gen_cheri_break_loadlink(out_addr, memop);
```

Comment:
Maybe we should do the `gen_cheri_break_loadlink` unconditionally (i.e. even for MIPS without CHERI). That would allow reverting https://github.com/CTSRD-CHERI/cheritest/commit/3d5146e450a9beb7c688763a3ec18702f3bc344d and https://github.com/CTSRD-CHERI/cheritest/commit/494c0dd2354215e3f53475d36fc1319e9000dc23

Reply:
OK; here's a version that tries to do just that.  This continues to pass `cheritest`.